### ISSUE TITLE
array.prototype.filter - 'not loaded: define-properties'

### ISSUE THREAD

**Issue by jeremy-holt** [twbs/bootstrap#34060]
I'm using aurelia v1.0, bootstrap 5.0.1
I ran yarn upgrade and am now getting an error `not loaded: define-properties`.
I've been able to track the error down to `array.prototype.filter` which is installed only by bootstrap 5.0.1
I'm guessing that the error is in the first line of index.js
```
'use strict';

var define = require('define-properties');
var RequireObjectCoercible = require('es-abstract/2020/RequireObjectCoercible');
var callBound = require('call-bind/callBound');

var implementation = require('./implementation');
var getPolyfill = require('./polyfill');
var polyfill = getPolyfill();
var shim = require('./shim');

var $slice = callBound('Array.prototype.slice');

// eslint-disable-next-line no-unused-vars
var bound = function filter(array, callbackfn) {
	RequireObjectCoercible(array);
	return polyfill.apply(array, $slice(arguments, 1));
};
define(bound, {
	getPolyfill: getPolyfill,
	implementation: implementation,
	shim: shim
});

module.exports = bound;
```
I have run npm define-properties but to no avail.

I have also cloned the array.prototype.filter project and can't even get it to run the tests

The project seems to have been updated a few days ago, but I can't find the previous version.

I should add that my project was running perfectly until this morning.

Window 10 pro 10.0.19043
Microsoft Edge Version 90.0.818.62

**Comment by jeremy-holt** [comment]
I solved this in the end by reverting to the old versions:
```
yarn add array.prototype.filter@0.1.1
yarn add available-typed-arrays@1.0.2
```

**Comment by ljharb** [comment]
I'm pretty convinced this isn't a problem in `array.prototype.filter`; we can discuss further in https://github.com/es-shims/Array.prototype.filter/issues/2
